lib/sparrow/fcm/v1/supervisor.ex

```diff
@@ -0,0 +1,30 @@
+defmodule Sparrow.FCM.V1.Supervisor do
+  @moduledoc """
+  Main FCM supervisor.
+  Supervises FCM token bearer and pool supervisor.
+  """
+  use Supervisor
+
+  @spec start_link([Keyword.t()]) :: Supervisor.on_start()
+  def start_link(arg) do
+    Supervisor.start_link(__MODULE__, arg)
+  end
+
+  @spec init([Keyword.t()]) ::
+          {:ok, {:supervisor.sup_flags(), [:supervisor.child_spec()]}}
+  def init(raw_fcm_configs) do
+    [raw_fcm_config] = raw_fcm_configs
```

Comment:
I suggest putting this match as `def init([raw_fcm_config])` instead, so the intent is cleared. I spent a while looking through the code to answer "why are we assuming there is a single config in a list of configs?", while the answer really is "there's no list of configs" -- it's just a format in which `init` expects its arg.